pypdf/_page.py

```diff
@@ -1988,8 +1988,9 @@ def process_operation(operator: bytes, operands: List[Any]) -> None:
                 for op in operands[0]:
                     if isinstance(op, (str, bytes)):
                         process_operation(b"Tj", [op])
+                    # The space width may be smaller than the font width, so the width should be 95%.
                     if isinstance(op, (int, float, NumberObject, FloatObject)) and (
-                        (abs(float(op)) >= _space_width)
+                        (abs(float(op) / 0.95) >= _space_width)
```

Comment:
Can we change this to apply the scaling factor to the right (constant) side once instead? For now, each iteration requires an additional computation, while IMHO we would only need one.

Reply:
Thank you. I tried to fix it.